src/main/java/at/fh/joanneum/irfc/persistence/entiy/EventEntity.java

```diff
@@ -44,8 +47,8 @@ public class EventEntity {
   @JoinColumn(name = "fk_event_location", nullable = false)
   private EventLocationEntity eventLocation;
 
-//  @Column(name = "fk_category")
-//  private CategoryEntity eventLocation;
-
+  //  @Column(name = "fk_category")
```

Comment:
Add TODO with ticket number of delete commented code

Reply:
its Ok like this